packages/sync/src/config/SyncConfig.ts

```diff
@@ -77,6 +81,10 @@ export class SyncConfig implements DataSyncConfig {
     this.clientConfig = this.init(clientOptions);
   }
 
+  public async getStorage() {
+    this.storage = await new StorageWrapper(window.localStorage, "9E715A41D45BA00A3D410C23BE436DAA");
```

Comment:
I know, I've hardcoded the key for now. just makes it easier for testing and won't matter as this won't be merged